### PR TITLE
Misc tips, log and data services

### PR DESCRIPTION
**Pull request by jlevy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -401,6 +402,7 @@ This guide is about AWS, not DevOps or server configuration management in genera
 -	Heroku’s [**Twelve-Factor App**](http://12factor.net/) principles list some established general best practices for deploying applications.
 -	**Pets vs cattle:** Treat servers [like cattle, not pets](https://blog.engineyard.com/2014/pets-vs-cattle). That is, design systems so infrastructure is disposable. It should be minimally worrisome if a server is unexpectedly destroyed.
 -	The concept of [**immutable infrastructure**](http://radar.oreilly.com/2015/06/an-introduction-to-immutable-infrastructure.html) is an extension of this idea.
+-	Minimize application state on EC2 instances. In general, instances should be able to be killed or die unexpectedly with minimal impact. State that is in your application should quickly move to RDS, S3, or other data stores not on that instance.
```
Comment (ThanosBaskous) [inline README.md]: Worth carving out an exception for / noting the existence of / pointing to EBS?

**Comment by ThanosBaskous** [comment]
👍 Great additions!

File: README.md (hunk 1 above)
Comment (jlevy) [inline README.md]: Yeah. And EFS I suppose. If you have a specific suggestion let me know.

**Comment by jlevy** [comment]
Better I hope?

**Comment by ThanosBaskous** [comment]
👍 Thanks

**Pull request merged**